### PR TITLE
Time machine

### PR DESCRIPTION
**Pull request by 18epedersen**
I created a time machine.

### CODE REVIEW

**Comment by jenniferdai1** [comment]
looks good!

File: timemachine.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def timemachine(x):
```
Comment (stefanv) [inline timemachine.py]: Let's use PEP8 syntax and cal this `time_machine`.  Also add a docstring describing "x".

**Comment by Seth-Park** [comment]
great!

**Comment by 18epedersen** [comment]
:+1:

**Pull request merged**